src/lancer.ts

```diff
@@ -457,9 +455,31 @@ Hooks.on("renderCombatTrackerConfig", extendCombatTrackerConfig);
 
 // Disable token vision and fog exploration by default in scene config
 Hooks.on("preCreateScene", scene => {
-  scene.updateSource({ tokenVision: false, "fog.exploration": false });
+  scene.updateSource({ tokenVision: false, fog: { exploration: false } });
```

Comment:
Yeah I'd like to add back dotkey support but last time I did it I lagged out repos so bad. Probably some dumb data model stuff.